blueman/plugins/applet/SerialManager.py

```diff
@@ -35,7 +34,7 @@ class SerialManager(AppletPlugin):
                              "Upon device disconnection the script will be sent a HUP signal</span>")},
     }
 
-    scripts: Dict[str, Dict[str, Popen[Any]]] = {}
+    scripts: Dict[str, Dict[str, "Popen[Any]"]] = {}
```

Comment:
The `Any` import is technically unused now.

Reply:
mypy still wants it.

Reply:
Ok, depends on the context... 🙈